src/js/components/Graphs/NodeInfo/nodeInfo.style.js

```diff
@@ -17,6 +17,6 @@
 import styled from 'styled-components';
 
 export const NodeIndoContainer = styled.div`
-    width: 100%;
     padding: 15px;
+    height: calc(100% - 30px);
```

Comment:
Why `-30px`?

Reply:
2 x padding 15px

Reply:
You could add `box-sizing: border-box` and I suspect you woulnd't need to substract this 30px.